tests/snappi_tests/dataplane/test_latency.py

```diff
@@ -0,0 +1,256 @@
+from tests.snappi_tests.dataplane.imports import *  # noqa F403
+sys.path.append('../test_reporting/telemetry')
+from snappi_tests.dataplane.files.helper import *  # noqa F403
+from metric_definitions import *   # noqa E402
+from reporter_factory import TelemetryReporterFactory  # noqa F403
+from metrics import GaugeMetric  # noqa E402
+
+common_labels = [
+    Point("Test_Info")
+    .tag("METRIC_LABEL_TESTBED", "TB-XYZ")
+    .tag("METRIC_LABEL_TEST_BUILD", "2024.1103")
+    .tag("METRIC_LABEL_TEST_CASE", os.path.basename(__file__))
+    .tag("METRIC_LABEL_TEST_FILE", os.path.basename(__file__))
+    .tag(
+        "METRIC_LABEL_TEST_JOBID",
+        f"{os.path.basename(__file__)}_{datetime.now():%Y%m%d_%H%M%S}"
+    )
+]
+
+reporter = TelemetryReporterFactory.create_periodic_metrics_reporter(common_labels)
+
+logger = logging.getLogger(__name__)
+pytestmark = [pytest.mark.topology("tgen")]
+
+# Test Parameters
+frame_sizes_rate = [
+    (66, 56.59),
+    (86, 69),
+    (128, 96.91),
+    (1500, 90),
+    (4000, 90),
+    (8000, 90),
+]
+num_iterations = 1
+oversubscription_percentage = 10
```

Comment:
we can be simple on this one. in the 2:1 or more:1 case, I don't think the 10% is a reasonable number.

all tx port should be 100% and goes to a single rx port. I have updated the HLD for this one.